src/components/Match/MatchHeader/MatchHeader.jsx

```diff
@@ -313,39 +325,55 @@ const MatchHeader = ({ match, user, strings }) => {
             </li>
             <li>
               <span>{strings.th_skill}</span>
-              {(match.skill) ? strings[`skill_${match.skill}`] : strings.general_unknown}
+              {match.skill
+                ? strings[`skill_${match.skill}`]
+                : strings.general_unknown}
             </li>
           </ul>
         </div>
       </div>
-      {!match.version &&
-        <Warning className="unparsed">
-          {strings.tooltip_unparsed}
-        </Warning>}
+      {!match.version && (
+        <Warning className="unparsed">{strings.tooltip_unparsed}</Warning>
+      )}
       <div className="matchButtons">
         <FlatButton
-          label={match.version ? strings.match_button_reparse : strings.match_button_parse}
+          label={
+            match.version
+              ? strings.match_button_reparse
+              : strings.match_button_parse
+          }
           icon={match.version ? <NavigationRefresh /> : <ActionFingerprint />}
           containerElement={<Link to={`/request#${match.match_id}`}>r</Link>}
         />
-        {match.replay_url &&
+        {match.replay_url && (
           <FlatButton
             label={strings.match_button_replay}
             icon={<FileFileDownload />}
             href={match.replay_url}
             target="_blank"
             rel="noopener noreferrer"
-          />}
+          />
+        )}
         <FlatButton
```

Comment:
did we want to put the sponsor button under a process.env config var for fast enable/disable?